mysql-test/suite/innodb/t/innodb_create_table_options.test

```diff
@@ -0,0 +1,281 @@
+--source include/have_innodb_max_16k.inc
+
+############################# ROW_FORMAT=COMPACT ###############################
+# Make sure that compact table creation works as usual.
+CREATE TABLE t_compact(a INT PRIMARY KEY) ENGINE=InnoDB ROW_FORMAT=COMPACT;
+SHOW CREATE TABLE t_compact;
+DROP TABLE t_compact;
+# Make sure that creating an compact table with COMPRESSION parameter set issues a warning and ignores this parameter
```

Comment:
Here and everywhere, no warning is issued in the result file.

Reply:
It's the same in the original patch.

Reply:
Should I remove these comments or add a warning?

Reply:
@hermanlee ^^